proto/panacea/oracle/v2alpha2/query.proto

```diff
@@ -23,7 +23,11 @@ service Query {
 
   // OracleRegistration returns a OracleRegistration details.
   rpc OracleRegistration(QueryOracleRegistrationRequest) returns (QueryOracleRegistrationResponse) {
-    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registrations/unique_id/{unique_id}/addresses/{address}";
```

Comment:
The meaning of this URL seems to be looking up address.
So, it was modified to look up OracleRegistration.

Reply:
Also, it is recommended to use hyphens rather than underscores in URLs.